todo/views.py

```diff
@@ -0,0 +1,75 @@
+from django.shortcuts import get_object_or_404, redirect
```

Comment:
I believe this imports are unused - check your code with some linters before pushing